aopy/analysis/connectivity.py

```diff
@@ -0,0 +1,238 @@
+import numpy as np
+from tqdm.auto import tqdm
+import multiprocessing as mp
+
+from . import base
+from .. import data as aodata
+from .. import precondition
+from . import accllr
+
+def get_acq_ch_near_stimulation_site(stim_site, stim_layout='Opto32', electrode_layout='ECoG244', 
+                                 dist_thr=1, return_idx=False):
+    '''
+    Get acquisition channels near a stimulation site. Warning: this function returns channel
+    numbers, which sometimes are 1-indexed. Set the return_idx flag to True to get the channel
+    indices as well.
+
+    Args:
+        stim_site: int, stimulation site
+        stim_layout: str, layout of stimulation sites
```

Comment:
What are the different layout options/are they in an sop?

Reply:
added links